.circleci/config.yml

```diff
@@ -48,10 +48,10 @@ workflows:
       - perf_test:
           requires:
             - cross_build
-          filters:
-            branches:
-              only:
-                - "master"
+#          filters:
```

Comment:
Need to change this back to run only on master right?

Reply:
Yes, before merge :+1: